internal/ui/index.go

```diff
@@ -0,0 +1,226 @@
+package ui
+
+const (
+	KeycloakInteractiveSuccessHTML = `<!DOCTYPE html>
```

Comment:
I think this could be `RedHatInteractiveSuccessHTML` since I don't think it should be limited to keycloak auth only.